### PR TITLE
Add kubewarden-1-20-release blogpost

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Fix https://github.com/kubewarden/kubewarden.io/issues/286

## Test

To test this pull request, you can run the following commands:

### CODE REVIEW

File: content/blog/2024/12/kubewarden-1-20-release.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+As always, we've addressed bugs and updated dependencies to ensure a smooth and
+reliable experience.
+
+Worth noting is a `kwctl` fix when dealing with local custom certificates for
+authenticating against OCI repositories when the user incorrectly passes a
+certificate in DER format (binary encoded) instead of PEM format (text, ASCII
+armored). Before, `kwctl` could stop with an error when trying to load the DER
+certificate. Now, `kwctl` has strenghtened PEM/DER recognition and `kwctl`
+informs better if one is using a DER certificate.
+
```
Comment (viccuad) [inline content/blog/2024/12/kubewarden-1-20-release.md:93]: I have left the new policies topic out for now.

Comment (viccuad) [reply content/blog/2024/12/kubewarden-1-20-release.md:94]: Added a commit with that section, contingent on rego-policies-library being ready.

Comment (viccuad) [reply content/blog/2024/12/kubewarden-1-20-release.md:93]: rebased, dropping the commit about the new policies for now.

File: content/blog/2024/12/kubewarden-1-20-release.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+>
+> This is of course reflected with a major version bump of the chart `version`.
+
+With v1.20, we have expanded the Kubewarden controller capabilities to
+configure OpenTelemetry integrations.
+
+Until v1.19 inclusive, one could only configure the kubewarden-controller chart to
+inject the Opentelemetry collectors [as a
+sidecar](https://opentelemetry.io/docs/collector/architecture/#a-nameopentelemetry-agentarunning-as-an-agent)
+on Pods in the Kubewarden stack.
```
[the remark this answers is not in the document]
Comment (viccuad) [reply content/blog/2024/12/kubewarden-1-20-release.md:27]:
```suggestion
into Pods in the Kubewarden stack.
```

**Review by jvanz — APPROVED** [review]

**Review by flavio — APPROVED** [review]
@viccuad squash the commits and merge it :partying_face:

**Pull request merged**